pkgs/tools/security/swtpm/default.nix

```diff
@@ -37,14 +37,19 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     libtpms
-    openssl libtasn1 libseccomp
-    fuse glib json-glib
+    openssl libtasn1
+    glib json-glib
     gnutls
+    python3
```

Comment:
Why is python needed?

Reply:
```
checking for gawk... /nix/store/agdkw5i65glh5zzisvjgw5qg443b5013-gawk-5.1.1/bin/gawk
checking for socat... /nix/store/8pksrxn0xm0vkl70niq0y5msxkfk29jm-socat-1.7.4.3/bin/socat
checking for base64... /nix/store/6pjf7rxdnb136a7cn9yj2mhy1wqm1qhq-coreutils-9.0/bin/base64
checking for cp... /nix/store/6pjf7rxdnb136a7cn9yj2mhy1wqm1qhq-coreutils-9.0/bin/cp
checking for a Python interpreter with version >= 3.3... none
configure: error: no suitable Python interpreter found
error: builder for '/nix/store/rly3222b9sb3xm27q3y6wjjl1ddiwqvm-swtpm-0.7.1.drv' failed with exit code 1;
       last 10 log lines:
       > checking for gnutls_load_file in -lgnutls... yes
       > checking for gnutls/abstract.h... yes
       > checking which PCR banks to activate by default... sha256
       > checking for expect... /nix/store/40qh485b48kcmicp3z4prwnk4r7n2rh0-expect-5.45.4/bin/expect
       > checking for gawk... /nix/store/agdkw5i65glh5zzisvjgw5qg443b5013-gawk-5.1.1/bin/gawk
       > checking for socat... /nix/store/8pksrxn0xm0vkl70niq0y5msxkfk29jm-socat-1.7.4.3/bin/socat
       > checking for base64... /nix/store/6pjf7rxdnb136a7cn9yj2mhy1wqm1qhq-coreutils-9.0/bin/base64
       > checking for cp... /nix/store/6pjf7rxdnb136a7cn9yj2mhy1wqm1qhq-coreutils-9.0/bin/cp
       > checking for a Python interpreter with version >= 3.3... none
       > configure: error: no suitable Python interpreter found
       For full logs, run 'nix log /nix/store/rly3222b9sb3xm27q3y6wjjl1ddiwqvm-swtpm-0.7.1.drv'.
```

Reply:
I think it's only for testing, but since autoconf looks for it, I assume it still needs to be there during the build phase

Reply:
it checks for it, but it's only used in tests.

Reply:
Fixed with 91eeb2e -- moved to `nativeBuildInputs` and removed from `checkInputs`.